<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReverseEngineerWizardWelcomeForm</class>
 <widget class="QWidget" name="ReverseEngineerWizardWelcomeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabWidgetPage1" native="true">
      <attribute name="title">
       <string>DB</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Host</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txtDatabaseHost"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="txtPort">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>User</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="txtDatabaseUser"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="txtDatabasePassword">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblDatabaseName">
         <property name="text">
          <string>Database</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="txtDatabaseName">
         <property name="echoMode">
          <enum>QLineEdit::Normal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabWidgetPage2" native="true">
      <attribute name="icon">
       <iconset resource="../dbm.qrc">
        <normaloff>:/images/actions/images/small/sqlite-icon.png</normaloff>:/images/actions/images/small/sqlite-icon.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Sqlite</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Database</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="txtSqliteFilename"/>
         </item>
         <item>
          <widget class="QToolButton" name="btnBrowseForFile">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string>Create a new database</string>
           </property>
           <property name="statusTip">
            <string>Create a new database</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../dbm.qrc">
             <normaloff>:/images/actions/images/big/folder_yellow.png</normaloff>:/images/actions/images/big/folder_yellow.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Version</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cmbSqliteVersion">
         <item>
          <property name="text">
           <string>Version 3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Version 2</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../dbm.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnBrowseForFile</sender>
   <signal>clicked()</signal>
   <receiver>ReverseEngineerWizardWelcomeForm</receiver>
   <slot>onBrowseFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBrowseFile()</slot>
 </slots>
</ui>
